discovery-provider/src/queries/get_health.py

```diff
@@ -183,7 +184,8 @@ def get_health(args, use_redis_cache=True):
         "git": os.getenv("GIT_SHA"),
         "trending_tracks_age_sec": trending_tracks_age_sec,
         "trending_playlists_age_sec": trending_playlists_age_sec,
-        **sys_info
+        **sys_info,
+        "number_of_cpus": number_of_cpus
```

Comment:
nit: would be nice to put this above sys_info so the unpack is the last argument